khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
I really wouldn't do it... to argue my points, I've found [this](http://python-future.org/unicode_literals.html#drawbacks), but it doesn't touch all of the issues we could experience.